### PR TITLE
## Release LogViewer 2.0.2

### PR DESCRIPTION
**Pull request by smflt-nhn**
### Updated
* Improved to change the size of the console log column according to the visible setting
* Improved to change flexibly according to the length of system information
* Popup size improved to change according to the message content

### CODE REVIEW

File: docs/LogViewer/ReleaseNotes.md [hunk 1]
```diff
@@ -6,6 +6,17 @@
 
 ### Date
 
+* 2020.03.31
+
+### Updated
+* Console 로그 컬럼이 보이는 설정에따라 크기가 변하도록 개선
```
Comment (SangYun-nhn) [inline docs/LogViewer/ReleaseNotes.md:12]: [MUST] 맞춤법 검사 결과에요

설정에따라 -> 설정에 따라

File: docs/LogViewer/ReleaseNotes.md [hunk 2]
```diff
@@ -6,6 +6,17 @@
...
+* System 정보 길이에 따라 유연하게 변하도록 개선
+* Popup 사이즈가 메세지 내용에따라 달라지도록 개선
```
Comment (SangYun-nhn) [inline docs/LogViewer/ReleaseNotes.md:14]: [MUST] 맞춤법 검사 결과에요

내용에따라 -> 내용에 따라

**Review by SangYun-nhn — APPROVED** [review]

File: docs/LogViewer/ReleaseNotes.md (hunk 1 above)
Comment (smflt-nhn) [reply docs/LogViewer/ReleaseNotes.md]: 적용하였습니다.

File: docs/LogViewer/ReleaseNotes.md (hunk 2 above)
Comment (smflt-nhn) [reply docs/LogViewer/ReleaseNotes.md]: 적용하였습니다.

**Review by spin-nhn — APPROVED** [review]

**Pull request merged**